build_scripts/build_mgard_cuda_turing.sh

```diff
@@ -9,6 +9,9 @@
 set -e
 set -x
 
+module load cuda/11.4
+module load cmake
+
```

Comment:
@ben-e-whitney This build script is for NVIDIA Turing GPU architectures (e.g., the sysytem76 machine in Scott's group) not for Volta GPUs on Summit. So there is no need to load modules.
For Summit, I have provided a dedicated build script `build_mgard_cuda_summit.sh`.

Reply:
Oh, OK. My mistake! I was trying to build everything. I'll revert this change. Should I keep the change to `build_scripts/build_mgard_serial.sh`?

Reply:
I think `build_mgard_cuda_summit.sh` is enough to build everything on Summit, so it is better to keep `build_mgard_serial.sh` unchanged for now.